### DISCUSSION TITLE
Where are the Manjaro and Raspbian-based builds?

### DISCUSSION THREAD

**Discussion by erikjber** [OpenVoiceOS/ovos-buildroot#180] [Q&A] [answered]
I can't find the image files for the Manjaro and Raspbian builds. When I go to the [OVOS homepage](https://openvoiceos.org/) and click "downloads" I am taken to a section with links to the images, but the links are all 404.

**Comment by j1nx** [comment]
We need some help to get all the latest images that where there from you people as users as they got lost during the server / domain change a week ago.

We are very sorry about that, but if anyone can share some temporarily links, we can download and re-upload them again.

**Comment by ChanceNCounter** [comment]
To elaborate, a former member of the OVOS board, who was the custodian of our server infrastructure while we're incorporating, quit a few weeks ago (the OVOS Foundation won't be able to own property until it's fully incorporated.) When the infrastructure was turned over, a couple things were missed, including the Manjaro and Raspian download server.

If nothing else, a newer version of the Raspbian image will be uploaded on our infra ASAP. The Manjaro version might be retired; we will have more information when a decision has been reached.

**Comment by arslin1** [comment]
Hello-

New to this type of device. Looks like I need this project for my Pi to work... (poke *** poke). I look forward to the build's availability soon. :) 

-M

**Comment by builderjer** [comment]
I have a new download page available for the raspbian-ovos images.
Just as a warning, these images are still WAY testing.  Please leave feedback on the [issue page](https://github.com/OpenVoiceOS/raspbian-ovos/issues) in the [raspbian-ovos repo](https://github.com/OpenVoiceOS/raspbian-ovos)

[Download images here](https://ovosimages.ziggyai.online/raspbian/)

**Comment by JarbasAl** [comment, marked as answer]
Manjaro images no longer exist

raspOVOS downloads here https://ovosimages.ziggyai.online/raspbian/
